packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -38,7 +38,26 @@ privacy = Privacy Notice
 
 ## plan details
 product-plan-details-heading = Let's set up your subscription
-product-plan-details-amount = { $productName } for ${ $amount } per { $interval }
+
+day-based-plan-details-amount = { $intervalCount ->
+  [one] { $productName } billed ${ $amount } daily
+  *[other] { $productName } billed ${ $amount } every {$intervalCount} days
+}
+
+week-based-plan-details-amount = { $intervalCount ->
+  [one] { $productName } billed ${ $amount } weekly
+  *[other] { $productName } billed ${ $amount } every {$intervalCount} weeks
+}
+
+month-based-plan-details-amount = { $intervalCount ->
+  [one] { $productName } billed ${ $amount } monthly
+  *[other] { $productName } billed ${ $amount } every {$intervalCount} months
+}
+
+year-based-plan-details-amount = { $intervalCount ->
+  [one] { $productName } billed ${ $amount } yearly
+  *[other] { $productName } billed ${ $amount } every {$intervalCount} years
```

Comment:
This is probably pretty unlikely, but do we want to conditionally pluralize the unit in case `$intervalCount` is ever `1`? e.g. `every 1 months` could be awkward looking.

Reply:
If `$intervalCount` is 1, then it will use the `[one]` case and be: `... billed $5.00 yearly`. You can see the verification in the tests. There is a test for equal to one and greater than 1 for each of the translations added.

Reply:
https://github.com/mozilla/fxa/pull/4573/files#diff-8466bb64c869abe0b93953b4ce769fe3R246

Reply:
Ahh, got it. I misunderstood how this was set up. Carry on!